src/Components/People.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import axios from "axios"
+
+export default class People extends Component {
+    constructor() {
+        super()
+        this.state = {
+            searchCharacter: "",
+            currentMovie: {},
+            isError: false,
+        }
+    }
+
+    handleChange = (e) => {
+        this.setState({searchCharacter: e.target.value})
+    }
+
+    handleSubmit = async (e) => {
+        e.preventDefault();
+        const { searchCharacter } = this.state
+        console.log(searchCharacter)
+        
+        try {
+            const { data } = await axios.get(`https://ghibliapi.herokuapp.com/people`)
+            this.setState({currentMovie: data, searchCharacter: "", isError: false})
```

Comment:
The data you get back from this api call is an array of characters... not just a single character. Try writing some code to find the character whose name matches `searchCharacter`, and then setState with only the matching character.